meta/main.yml

```diff
@@ -5,14 +5,14 @@ galaxy_info:
   description: An Ansible role for installing Open Trip Planner.
   company: Azavea Inc.
   license: Apache
-  min_ansible_version: 1.5
+  min_ansible_version: 2.0
   platforms:
   - name: Ubuntu
     versions:
-    - trusty
+    - xenial
```

Comment:
Is Oracle Java working properly now on Ubuntu server? Last I tried working with it, it was missing some dependencies apparently present on Ubuntu desktop; I wasn't able to track them all down. It would install without error but couldn't run OTP properly.